tests/mongodb_create_connect.robot

```diff
@@ -12,20 +12,29 @@ Test Teardown       Close Browser
 
 *** Test Cases ***
 Scenario: Error Message To Select Valid Namespace For Provider Account Creation
-    [Tags]    smoke
-    When User Filters Project Other Than redhat-dbaas-operator And Navigates To Database Access Page
+    [Tags]    smoke    RHOD-45
+    Skip
```

Comment:
Can you please confirm whether we move this test to different robot file or skipping for now?

Reply:
Before raising the PR, I planned to save the output log for mongo and crunchy separately and wanted them to be clean, I was hoping to discuss with the team before moving this test case to another Test Suite
So in the dilemma of not moving the Test Case and also wanting the logs to be clean, I just skipped it for the time being.
Please rest assured this is a very temporary change and will be amended with later PRs

Reply:
Thanks for the clarification